### ISSUE TITLE
quotes parsing error

### ISSUE THREAD

**Issue by mathematikoi** [jspython-dev/jspython#34]
the following gets interpreted successfully, although the syntax is wrong!
```
const {jsPython} = require("jspython-interpreter");

const script = `
    'hello world
`

jsPython()
  .evaluate(script)
  .then((result) => {
    console.log(result);
  })
  .catch((error) => {
    console.log(error);
  });

```

**Comment by ppaska** [comment]
Another use case:

```python
t = "some str ...
return t
```

**Comment by ppaska** [comment]
released v2.1.12
